packages/application-generic/src/services/bull-mq/bull-mq.service.ts

```diff
@@ -81,7 +83,30 @@ export class BullMqService {
     };
   }
 
+  private getQueueBaseOptions(): QueueBaseOptions {
+    const inMemoryProviderConfig: IRedisProviderConfig =
+      this.inMemoryProviderService.inMemoryProviderConfig;
+
+    const bullMqBaseOptions = {
+      connection: {
+        db: Number(process.env.REDIS_DB_INDEX),
+        port: inMemoryProviderConfig.port,
+        host: inMemoryProviderConfig.host,
+        password: inMemoryProviderConfig.password,
+        connectTimeout: inMemoryProviderConfig.connectTimeout,
+        keepAlive: inMemoryProviderConfig.ttl,
+        family: inMemoryProviderConfig.family,
+        keyPrefix: inMemoryProviderConfig.keyPrefix,
+        tls: inMemoryProviderConfig.tls,
+      },
```

Comment:
Now we feed the BullMq config from the different provider configurations we might inject for each case.